cmds/admin_server/job/storage.go

```diff
@@ -0,0 +1,31 @@
+package job
+
+import (
+	"time"
+
+	"github.com/linuxboot/contest/pkg/types"
+	"github.com/linuxboot/contest/pkg/xcontext"
+)
+
+// DB wraps a job database
+type Storage interface {
+	GetTags(ctx xcontext.Context, tagPattern string) ([]Tag, error)
+	GetJobs(ctx xcontext.Context, projectName string) ([]Job, error)
+}
+
+// Tag contains metadata about jobs under a given tag (project)
```

Comment:
as i said before, `tag` is not `project`. you can implement the projects concept with tags, but theyre not identical
```suggestion
// Tag contains metadata about jobs under a given tag
```